### ISSUE TITLE
New Twitter DOM parsing

### ISSUE THREAD

**Issue by Jolg42** [malteserteresa/stop-it#8]
We could subscribe to this issue and share what we find, if we find a solution ;-)

https://github.com/sindresorhus/refined-twitter/issues/141

**Comment by linahanner** [comment]
@Jolg42 What's the issue in question? I wasn't listening at the time that you were discussing it. Could you edit the issue description? 😄

**Comment by jancborchardt** [comment]
It’s not a big issue and we resolved it at the meetup. :) There is a code piece to get the text content of all Tweets on a page (this for the current Twitter):
```
document.querySelectorAll('.tweet-text').forEach(function(tweets) {
  console.log(tweets.innerText);
});
```

Twitter is currently redesigning (you can try out "the new Twitter") if you click on your user icon on the top right and in that menu is the last entry "Try the new Twitter").
Since structure changed, the command for the new Twitter is:
```
document.querySelectorAll('article [lang]').forEach(function(tweets) {
  console.log(tweets.innerText);
});
```

So basically the same, just now instead of `.tweet-text` the selector is `article [lang]`. That is because every Tweet is an `article`, and the text of it is in an element which has a `lang` attribute. It could even target only English tweets first with `article [lang="en"]`. :)
